Sources/Diagnose/ReproducerBundle.swift

```diff
@@ -47,14 +58,14 @@ func makeReproducerBundle(for requestInfo: RequestInfo, toolchain: Toolchain, bu
     )
   }
   for compilerArg in requestInfo.compilerArgs {
-    // Copy all files from the compiler arguments into the reproducer bundle.
-    // Don't include files in Xcode (.app), Xcode toolchains or usr because they are most likely binary files that aren't user specific and would bloat the reproducer bundle.
-    if compilerArg.hasPrefix("/"), !compilerArg.contains(".app"), !compilerArg.contains(".xctoolchain"),
-      !compilerArg.contains("/usr/")
-    {
-      let dest = URL(fileURLWithPath: bundlePath.path + compilerArg)
-      try? FileManager.default.createDirectory(at: dest.deletingLastPathComponent(), withIntermediateDirectories: true)
-      try? FileManager.default.copyItem(at: URL(fileURLWithPath: compilerArg), to: dest)
+    if compilerArg.hasPrefix("/") {
+      copyPathToBundleIfNecessary(compilerArg, bundlePath: bundlePath)
+    } else if let firstEqual = compilerArg.firstIndex(of: "=") {
+      // Copy files from eg. `-fmodule-map-file=/path/to/module.modulemap`
+      let possiblePath = String(compilerArg[compilerArg.index(after: firstEqual)...])
+      if possiblePath.hasPrefix("/") {
+        copyPathToBundleIfNecessary(possiblePath, bundlePath: bundlePath)
+      }
```

Comment:
Do we care about cases like `-I/foo/bar`?

Reply:
Let’s just look for the first `/` to start the path. Eventually I’ll need to think about how this should work on Windows as well but right now the entire diagnose bundle pretty heavily relies on oslog scraping, which is only applicable for macOS anyway.